lesson14/style.css

```diff
@@ -50,6 +52,22 @@ padding: 30px;
 border-radius: 8px;
 }
 
+.modal-body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.modal-body #js-btn-fetch{
+  margin-top: 20px;
+}
+
+#js-input-number {
+  width: 100px;
+  padding: 15px 25px;
+  font-size: 18px;
+}
```

Comment:
this class name has `js-` prefix.
separate the css class name and the name used in js file

```
<div id="js-input-number" class="input-number">
```

Reply:
Thank you.
I fixed it.